### DISCUSSION TITLE
blog/2022/12/08/log10-natural-log-scales-ggplot/index

### DISCUSSION THREAD

**Discussion by giscus** [andrewheiss/ath-quarto#53] [Blog comments]
# blog/2022/12/08/log10-natural-log-scales-ggplot/index

Use the {scales} R package to automatically adjust and format x- and y-axis scales to use log base 10 and natural log values

https://www.andrewheiss.com/blog/2022/12/08/log10-natural-log-scales-ggplot/index.html

**Comment by briatte** [comment]
See also [this method to replicate "base" breaks with ggplot2](https://stackoverflow.com/questions/14255533/pretty-ticks-for-log-normal-scale-using-ggplot2-dynamic-not-manual/14255748#14255748).

**Comment by mkmiecik14** [comment]
Andrew, absolutely beautiful job here. I appreciated the post and love your website. Thank you!
